app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -232,10 +252,10 @@ Here is a visualization of the Interpolator using the example from [Timeline](#e
 ## AnimationTimer
 
 The [AnimationTimer](javafxdoc:AnimationTimer) abstract class provides the lowest level option to create an animation.
-The `handle(long now)` method gets called in each frame while it is active. The timestamp `long now` is the nanoseconds time of the current frame and will be the same for all `AnimationTimer` called during that frame.
-Additionally, the `AnimationTimer` adds the `start()` and `stop()` to handle the lifetime of the animation.
+The `handle(long now)` method gets called in each frame while it is active. The timestamp `now` is the time of the current frame in nanoseconds and will be the same for all `AnimationTimer`s called during that frame.
+Additionally, the `AnimationTimer` adds `start()` and `stop()` methods to handle the lifetime of the animation.
 
-**Note:** The handle method will be called on the **JavaFX Application Thread** and thus shouldn't do heavy computations.
+**Note:** The `handle` method will be called in the **JavaFX Application Thread** and thus shouldn't do heavy computations.
```

Comment:
Just a note This isn't really about _computation_ but more about the time the operation takes. It could as well be I/O (e.g. network calls) that slow it down.

Reply:
I agree. Maybe "should complete quickly"? Or maybe with a bit of extra explanation along the lines of "many such operations need to be executed within a frame and for a smooth 30fps that means everything the transition and all of JavaFX need to compute for it needs to be done within 33ms" ? After either of those options a follow-up sentence could be "That excludes heavy computations and of course all blocking operations like I/O."